addons/help/XEH_preStart.sqf

```diff
@@ -38,32 +38,38 @@ private _credits = [];
     _credits pushBack format ["<font color='#bdcc9c'>%1%2 by %3</font>", _name, _version, _author];
 } forEach _addons;
 
-_credits = (_credits arrayIntersect _credits) joinString "<br/>";
+_credits = _credits arrayIntersect _credits;
+
+_credits sort true;
+
+_credits = _credits joinString "<br/>";
 
 uiNamespace setVariable [QGVAR(credits), compileFinal str _credits];
 
 // mods
 private _mods = "true" configClasses (configFile >> "CfgPatches") apply {configSourceMod _x};
-_mods = (_mods arrayIntersect _mods select {!isNumber (configfile >> "CfgMods" >> _x >> "appId")}) - [""];
+_mods = ((_mods arrayIntersect _mods) select {!isNumber (configfile >> "CfgMods" >> _x >> "appId")}) - [""];
 
 _mods = _mods apply {
     private _entry = configfile >> "CfgMods" >> _x;
 
-    private _name = getText (_entry >> "name") call CBA_fnc_sanitizeHTML;
+    private _name = getText (_entry >> "name");
 
     if (isClass _entry) then {
         _x = format ["    <font color='#cc9cbd'>%1 - %2</font>", configName _entry, _name];
 
         if (isText (_entry >> "description")) then {
-            private _description = getText (_entry >> "description") call CBA_fnc_sanitizeHTML;
+            private _description = getText (_entry >> "description");
 
             _x = _x + format ["<br/>%1<br/>", _description];
         };
     };
 
-    _x
+    _x call CBA_fnc_sanitizeHTML
```

Comment:
I think we want to revert this as it's sanitizing the font tag?
![image](https://github.com/user-attachments/assets/a8ba73b4-4817-4e45-b02b-b8412dda8879)

Reply:
Never knew that was a thing, because for me steam make a folder named `@CBA_A3`, which never make it do the fancy name and description. Imo that's broken.

Reply:
looks like https://community.bistudio.com/wiki/modParams

is designed to take what `configSourceMod` gives us, which could be 
anything between "cba" or "@CBA_A3" or "450814997"

interesting comment on that cmd
> Later versions of PboProject and its friends with arma as the intended engine, will never allow a cfgmods to appear in a pbo. Instead, it will scribble it out to a mod.cpp in the mods\ folder that this pbo was intended for.

so I guess for most people descrption won't show, but it's not a big eal

Reply:
> so I guess for most people descrption won't show, but it's not a big eal

Could use `modParams`' `overview` or `tooltip` if they exists and are not `""`, but I don't really care either way.